packages/app-harness/renative.json

```diff
@@ -105,7 +110,8 @@
                                         "tag": "activity",
                                         "android:name": "com.ahmedadeltito.photoeditor.PhotoEditorActivity",
                                         "children": []
-                                    }
+                                    },
+                                    { "tag": "activity", "android:name": "com.yalantis.ucrop.UCropActivity" }
```

Comment:
@ElenaDiachenko so does this change mean you also fixing something else apart from just pure renaming of activity? if so can you reference issue or describe it in the PR